Ruth/Chapter_1/display_eval.py

```diff
@@ -0,0 +1,3 @@
+(6 + 4 * 9) # this will not display the output on the prompt
```

Comment:
Is this line suppose to be `6 + 4 * 9` or `(6 + 4 * 9)`? Maybe you need to fix this?